.circleci/config.yml

```diff
@@ -128,6 +131,11 @@ workflows:
   version: 2
   build_and_test:
     jobs:
+      - prodsec/secrets-scan:
+          name: Scan repository for secrets
+          context:
+            - snyk-bot-slack
+          channel: team-hammerhead
```

Comment:
`hammerhead-alerts` would be a more appropriate channel

Reply:
thank you, I will update this and the other contributions